file_upload_handler/src/server.js

```diff
@@ -0,0 +1,67 @@
+import express from 'express'
+import multer from 'multer'
+
+const app =express();
+
+const storage = multer.diskStorage({
```

Comment:
TLDR should remove this

Reply:
Did a little bit of reading
https://help.heroku.com/K1PPS2WM/why-are-my-file-uploads-missing-deleted
Apparently we do have access to the file system of or dyno, but those files get deleted every time the dyno restarts.
I'm not sure we could actually use this though, since we need to pass these files between our different apps (servers)